### DISCUSSION TITLE
What's the difference between this package and gorm or https://github.com/uptrace/bun?

### DISCUSSION THREAD

**Discussion by frederikhors** [go-rel/rel#240] [Q&A] [answered]
What's the difference between this package and gorm or https://github.com/uptrace/bun?

**Comment by Fs02** [comment, marked as answer]
I've never used bun, but I think all of this package have the same goal, but built using different philosophy/style.
I personally decided to built REL because I'm not satisfied with gorm.

I think the biggest difference that REL have is: 
- The repository is designed in a way it's testable, but still have a neat and easy to use api ([motivation](https://go-rel.github.io/introduction/#easy-to-test))
- The transaction API is seamless, which something that I only found in REL as of now ([motivation](https://go-rel.github.io/introduction/#seamless-transactions))
